.circleci/config.yml

```diff
@@ -28,6 +32,7 @@ jobs:
     steps:
       - checkout
       - run: *deps-install
+      - run: *plsma-install
```

Comment:
I seriously have no idea why this line is necessary. AFAIK `steps` are not done in parallel.

Reply:
I think that's precisely why it needs to be run like this. Steps are run sequentially so the dependency installation step gets to complete and then numpy exists at setup time. Whereas this:
```
|
     python3 -m venv venv
     . venv/bin/activate
-    pip install -q --user -r requirements/automated-documentation-tests.txt
-    pip install -q --user .
```
doesn't run as separate commands but rather as one large command. (The entire thing only has one unix dollar sign associated with it when looking at the circleci output rather than a separate dollar sign for each line.)